.github/workflows/main.yml

```diff
@@ -533,7 +691,7 @@ jobs:
         run: |
           cd easy-indysdk && standard
 
-  publish-images:
+  docker-publish-images-all:
```

Comment:
I don't think `all` is accurate here (two other images are published in separate jobs)

Reply:
I don't see this addressed :)